template/scripts/sample-script.js

```diff
@@ -3,7 +3,7 @@
 //
```

Comment:
I would turn this into TS - see https://github.com/Developer-DAO/token-airdrop-whitelist/blob/master/scripts/deploy.ts

Reply:
That's a good idea. I'm thinking of creating a separate issue & PR for that.